src/Advertising.js

```diff
@@ -50,52 +55,15 @@ export default class Advertising {
     if (queue.length === 0) {
       return
     }
-    for (let i = 0; i < queue.length; i++) {
-      const { id, customEventHandlers } = queue[i]
-      Object.keys(customEventHandlers).forEach((customEventId) => {
-        if (!this.customEventCallbacks[customEventId]) {
-          this.customEventCallbacks[customEventId] = {}
-        }
-        return (this.customEventCallbacks[customEventId][id] = customEventHandlers[customEventId])
-      })
-    }
-    const divIds = queue.filter(({ id }) => id)
-    const selectedSlots = queue.map(({ id }) => {
-      return slots[id].gpt || outOfPageSlots[id]
-    })
+    this.setCustomEventCallbackByQueue(queue)
+    const { divIds, selectedSlots } = this.getDivIdsAndSlots(queue, outOfPageSlots, slots)
 
     if (isPrebidUsed) {
-      Advertising.queueForPrebid(
-        () =>
-          window.pbjs.requestBids({
-            adUnitCodes: divIds,
-            bidsBackHandler: () => {
-              window.pbjs.setTargetingForGPTAsync(divIds)
-              this.requestManager.prebid = true
-              this.refreshSlots(selectedSlots)
-            },
-          }),
-        this.onError
-      )
+      Advertising.queueForPrebid(this.getPbjFetchBidsCallback(divIds, selectedSlots), this.onError)
```

Comment:
Can we use "Pbjs" or "Prebid" instead of "Pbj" for the naming here?

Reply:
sure